go/cmd/vttablet/vttablet.go

```diff
@@ -74,7 +74,8 @@ func main() {
 	}
 
 	// creates and registers the query service
-	qsc := tabletserver.NewServer()
+	ts := topo.Open()
```

Comment:
I don't think the function is idempotent, currently. It looks to me like it opens an RPC connection.